src/libs/SidebarUtils.ts

```diff
@@ -125,16 +125,18 @@ function getOrderedReportIDs(
         // Normally, the spread operator would be used here to clone the report and prevent the need to reassign the params.
         // However, this code needs to be very performant to handle thousands of reports, so in the interest of speed, we're just going to disable this lint rule and add
         // the reportDisplayName property to the report object directly.
-        // eslint-disable-next-line no-param-reassign
-        report.displayName = ReportUtils.getReportName(report);
+        if (report) {
+            // eslint-disable-next-line no-param-reassign
+            report.displayName = ReportUtils.getReportName(report);
+        }
 
-        const isPinned = report.isPinned ?? false;
-        const reportAction = ReportActionsUtils.getReportAction(report.parentReportID ?? '', report.parentReportActionID ?? '');
-        if (isPinned || ReportUtils.requiresAttentionFromCurrentUser(report, reportAction)) {
+        const isPinned = report?.isPinned ?? false;
+        const reportAction = ReportActionsUtils.getReportAction(report?.parentReportID ?? '', report?.parentReportActionID ?? '');
+        if ((isPinned || ReportUtils.requiresAttentionFromCurrentUser(report, reportAction)) && report) {
             pinnedAndGBRReports.push(report);
-        } else if (report.hasDraft) {
+        } else if (report?.hasDraft) {
             draftReports.push(report);
-        } else if (ReportUtils.isArchivedRoom(report)) {
+        } else if (ReportUtils.isArchivedRoom(report) && report) {
```

Comment:
```suggestion
        } else if (report && ReportUtils.isArchivedRoom(report)) {
```

nit, makes more sense